### PR TITLE
PS-3313 When using proxy protocol, mysql must bind to an address inst…

### PR DESCRIPTION
**Pull request by patrickbirch**
…ead of 0.0.0.0 (5.6)

modified file:
flexibility/proxy_protocol_support

Rewrote first paragraph
added note to the variable

### CODE REVIEW

**Comment by patrickbirch** [comment]
waiting for comments or approval.

**Review by nastena1606 — APPROVED** [review]
LGTM

**Pull request merged**
